main.nf

```diff
@@ -478,9 +478,9 @@ process bowtie2 {
     module 'bowtie2'
     module 'samtools'
 
-    cpus 4
-    memory { 32.GB * task.attempt }
-    time { 24.h * task.attempt }
+    cpus 8
```

Comment:
If you're assigning 8 cores (presumably because you need the memory? You may as well get bowtie2 to use all of them with `-p 8`